Makefile

```diff
@@ -41,6 +41,14 @@ lint: generate
 generate: kustomize yq
 	scripts/generate.sh
 
+.PHONY: deploy
+deploy: generate
+	scripts/deploy-kubevirt-and-cdi.sh && KUBECTL=kubectl scripts/sync.sh
+
+.PHONY: functest
+functest:
+	cd tests && kubectl config view --flatten > /tmp/kubeconfig && KUBECONFIG=/tmp/kubeconfig go test -v -timeout 0 ./functests/... -ginkgo.v -ginkgo.randomize-all $(FUNCTEST_EXTRA_ARGS)
```

Comment:
What does writing to `/tmp/kubeconfig` first help with?

Reply:
Hi @0xFelix, thank you for your comment.
If I understood the code correctly, in order for the function tests to work effectively, the variable KUBECONFIG must point to a kubectl config file. Given this information, I have thought of a solution: dumping the config to a temporary file and then loading it as an environment variable for the tests.
I am open to any ideas, comments, suggestions, etc. How would you approach solving this?

Reply:
If you don't specify `KUBECONFIG` it should pick up the default `~/.kube/config` that in this case is also used by `kubectl`. Did it not work without `KUBECONFIG`?

Reply:
For example, in my case, when using minikube, it does not write the configuration to `~/.kube/config`.  Therefore, I came up with the idea of dumping the config to an extra file and changing `KUBECONFIG` only within the context of the function tests.

Reply:
Where is `kubectl config view --flatten` looking at in this case? IIUC without `KUBECONFIG` it will look at `~/.kube/config`?

Reply:
Solved in a more elegant way. @0xFelix you can take a look.